CHANGES.rst

```diff
@@ -1,13 +1,13 @@
 Change Log
 ==========
 
-Unreleased
-----------
+6.3.1 (2019/06/10)
+------------------
 
-**Changed**
+**Removed**
 
-* Removed ``FullnameMixin.kind``
-* Removed ``SubredditListingMixin.gilded()``
+* ``FullnameMixin.kind``
```

Comment:
Consider simply dropping this from the README here and in the 6.3.0 readme.

Reply:
alright.